tuf/src/database.rs

```diff
@@ -1314,4 +1449,89 @@ mod test {
 
         assert!(tuf.update_targets(&raw_targets).is_err());
     }
+
+    #[test]
+    fn good_metadata_update() {
```

Comment:
This test name could be more descriptive. What's "good" about this metadata update?

Reply:
done